lib/data/multipleBackendGateway.js

```diff
@@ -77,9 +147,36 @@ const multipleBackendGateway = {
                     }
                     cb();
                 });
+            } else if (client.clientType === 'aws_s3') {
+                client.headBucket({ Bucket: client.awsBucketName },
+                err => {
+                    if (err) {
+                        multBackendResp[location] = { error: err };
```

Comment:
Oh, I guess it could return bucket missing too, but the user gets the entire error back. Do you think they need more information?